cli/verbs.go

```diff
@@ -85,6 +85,23 @@ func run(requestor string, transport transport.Transport, stdout io.Writer) erro
 			if err != nil {
 				return err
 			}
+
+			//write the result into it's own file so that firmwareci
+			//can more easily parse the results
+			buffer.Reset()
+			if err := encoder.Encode(resp); err != nil {
+				return err
+			}
+
+			jobLog, err := os.Create("/logs/job.result")
```

Comment:
can we please specify the path via the config file?

Reply:
Understood why this needs to be in a special location. Alright.

Reply:
We could do it via an environment variable, that is set inside the container, and if it is not set default to a hardcoded file.